.flake8

```
@@ -0,0 +1,8 @@
+# This file is not used by anything in the repo.
+# It is here as a convenience to developers who use flake8 locally.
+
+[flake8]
+ignore =
+    E203  # whitespace before ':' - conflicts with black
+    E501  # line too long - black takes care of this for us
+    W503  # line break before binary operator - this is what black does
```

I don't believe this format doesn't work for flake8 6.0.0 and above due to stricter parsing of the config file. See https://github.com/pycqa/flake8/issues/1689#issuecomment-1326319235.

This should work instead:
```suggestion
    # whitespace before ':' - conflicts with black
    E203

    # line too long - black takes care of this for us
    E501

    # line break before binary operator - this is what black does
    W503
```

---

Oh, I wasn't aware of that -- thanks!